sources/logdog/src/error.rs

```diff
@@ -124,6 +124,15 @@ pub(crate) enum Error {
     #[snafu(display("Empty command."))]
     ModeMissing {},
 
+    #[snafu(display("Error parsing glob pattern '{}': {}", pattern, source))]
+    ParseGlobPattern {
+        pattern: String,
+        source: glob::PatternError,
+    },
+
+    #[snafu(display("Glob pattern missing"))]
+    PatternMissing {},
```

Comment:
I am not sure if I understood you comment, there is no file. This error will occur when someone gives "glob", basically forgot the pattern part in conf.

Reply:
Yeah these 'conf' files are pulled in statically at compile time.

Reply:
Ah, then most users would be using what we distribute, so they couldn't change it.  Maybe we should just expand this error message, then.  If we mess up testing, and users only see "Glob pattern missing", it seems like it could be a little confusing.  How about something like ``The logdog configuration has a "glob" line with no glob instructions.  Please check `sources/logdog/conf` and/or file an issue with Bottlerocket.``?

Reply:
I feel the message is too verbose. We are creating the conf file, so ideally we would never see this error in actual run. 
Still if it occurs I feel message "The logdog configuration has a "glob" line with no glob instructions" is good enough.